### PR TITLE
Added a public method to get the name from dump, changed the comment of the public method, and added the icon style of label.

### PR DESCRIPTION
**Pull request by chichinohaha**
Re: cocos-creator/3d-tasks#
fixed https://github.com/cocos-creator/3d-tasks/issues/6564
Changelog:
 * 
image

fixed https://github.com/cocos-creator/3d-tasks/issues/6564

### CODE REVIEW

File: editor/inspector/components/label.js [hunk 1]
```diff
@@ -0,0 +1,204 @@
+const { getNameFromDump, setHidden, isMultipleInvalid } = require('../utils/prop');
+const { template, $, update } = require('./base');
+
+exports.template = template;
+exports.$ = $;
+exports.update = update;
+exports.style = /* css */`
```
Comment (arsen2010) [inline editor/inspector/components/label.js:7]: useless , remove it better.

File: editor/inspector/components/label.js [hunk 2]
```diff
@@ -0,0 +1,204 @@
...
+    border: calc(var(--size-normal-border) * 1px) solid var(--color-normal-border);
+    background: var(--color-default-fill);
+    border-radius: calc(var(--size-normal-radius) * 1px);
+}
+
+div.fontStyle.invalid {
+    background: var(--color-default-fill);
+}
+
+div.fontStyle.select {
```
Comment (arsen2010) [inline editor/inspector/components/label.js]: div selector is useless

File: editor/inspector/components/label.js [hunk 3]
```diff
@@ -0,0 +1,204 @@
...
+    background: var(--color-default-fill-emphasis);
```
Comment (arsen2010) [inline editor/inspector/components/label.js]: background-color  would be more exactly.

File: editor/inspector/components/label.js [hunk 4]
```diff
@@ -0,0 +1,204 @@
...
+                const label = document.createElement('ui-label');
+                label.setAttribute('slot', 'label');
+                label.setAttribute('value', getNameFromDump(dump));
+                const content = document.createElement('ui-tab');
+                content.setAttribute('slot', 'content');
+                content.addEventListener('change', (event) => {
+                    if (event.target.value !== -1) {
+                        dump.value = event.target.value;
+                        if (dump.values) {
+                            dump.values = dump.values.map(() => dump.value);
```
Comment (arsen2010) [inline editor/inspector/components/label.js]: use forEach would be better

File: editor/inspector/components/label.js [hunk 5]
```diff
@@ -0,0 +1,204 @@
...
+
+                for (let index = 0; index < dump.enumList.length; index++) {
+                    const element = dump.enumList[index];
+                    const image = document.createElement('ui-image');
+                    const button = document.createElement('ui-button');
+                    const iconName = element.name.toLocaleLowerCase();
+                    if (iconName !== 'center') {
+                        image.setAttribute('value', `packages://scene/static/icons/align-${iconName}.png`);
+                    } else {
+                        image.setAttribute('value', `packages://scene/static/icons/align-h-${iconName}.png`);
```
Comment (arsen2010) [inline editor/inspector/components/label.js:69]:
```
if (iconName === 'center') {

} else if () {
} else {
}

```
would be better

File: editor/inspector/components/label.js [hunk 6]
```diff
@@ -0,0 +1,204 @@
...
+                    }
+                    image.style.height = '20px';
+                    image.style.width = '22px';
+                    image.style.verticalAlign = 'middle';
```
Comment (arsen2010) [inline editor/inspector/components/label.js:73]: style use css selector, can save moe code

File: editor/inspector/components/label.js [hunk 7]
```diff
@@ -0,0 +1,204 @@
...
+                    image.setAttribute('fill', true);
+                    image.setAttribute('tooltip', Editor.I18n.t(dump.tooltip.replace('i18n:', '')));
```
Comment (arsen2010) [inline editor/inspector/components/label.js]: unnecessary use ".replace('i18n:', '')"

File: editor/inspector/components/label.js [hunk 8]
```diff
@@ -0,0 +1,204 @@
...
+                    });
+                    content.appendChild(div);
+                }
+                prop.appendChild(content);
+                return prop;
+            },
+            update(element, dump) {
+                const parent = element.querySelector('[slot="content"]');
+                for (let index = 0; index < parent.childNodes.length; index++) {
+                    const div = parent.childNodes[index];
```
Comment (arsen2010) [inline editor/inspector/components/label.js]: depend on `const styles = ['isBold', 'isItalic', 'isUnderline'];` to loop select div element ,would be more stable.

**Review by arsen2010 — APPROVED** [review]

**Review by VisualSJ — APPROVED** [review]

**Pull request merged**